tests/cpython-tests/Makefile

```diff
@@ -30,8 +30,9 @@ TEST_RUN_INDIVIDUAL_3_10=$(shell cat test_config_v3.10.2/tests.passed.individual
 export TIMEOUT=3600
 
 all: ext2fs3.10 
```

Comment:
Should this be `all: ext2fs3.9` or just `all`?

Reply:
<sigh> I'll remove it again. Must have hit undo at some point.